guardian/ctypes.py

```diff
@@ -0,0 +1,31 @@
+from django.contrib.contenttypes.models import ContentType
+
+
+def get_ctype_from_polymorphic(obj):
```

Comment:
Yeah. I was hoping to get general feedback on the approach before completing the full implementation with tests.